projects/admin/src/app/components/user-profile/user-profile.component.ts

```diff
@@ -26,7 +26,8 @@ import { translations } from '../../translations';
   styleUrl: './user-profile.component.scss',
 })
 export class UserProfileComponent {
-  translations = translations.language;
+  translations = translations.general;
+  translation = translations.language;
```

Comment:
if we need more than one then use the generic one 
```suggestion
  translations = translations;
```

or 2 
```suggestion
  generalTranslations = translations.general;
  languageTranslations = translations.language;
```
but I prefer 1 option